### ISSUE TITLE
[v5] Attachments was lost after migration from 5.3.5 to 5.5 version

### ISSUE THREAD

**Issue by v1-wizard** [reportportal/reportportal#1507]
**Describe the bug**
We don't see any attachments on our instance and also see errors in logs:
```
2021-08-20 09:19:00.295 UTC [2907] STATEMENT:  select log0_.id as id1_9_0_, log0_.attachment_id as attachme8_9_0_, log0_.last_modified as last_mod2_9_0_, log0_.launch_id as launch_i9_9_0_, log0_.log_level as log_leve3_9_0_, log0_.log_message as log_mess4_9_0_, log0_.log_time as log_time5_9_0_, log0_.project_id as project_6_9_0_, log0_.item_id as item_id10_9_0_, log0_.uuid as uuid7_9_0_, attachment1_.id as id1_21_1_, attachment1_.content_type as content_2_21_1_, attachment1_.creation_date as creation3_21_1_, attachment1_.file_id as file_id4_21_1_, attachment1_.file_size as file_siz5_21_1_, attachment1_.item_id as item_id6_21_1_, attachment1_.launch_id as launch_i7_21_1_, attachment1_.project_id as project_8_21_1_, attachment1_.thumbnail_id as thumbnai9_21_1_, launch2_.id as id1_8_2_, launch2_.approximate_duration as approxim2_8_2_, launch2_.description as descript3_8_2_, launch2_.end_time as end_time4_8_2_, launch2_.has_retries as has_retr5_8_2_, launch2_.last_modified as last_mod6_8_2_, launch2_.mode as mode7_8_2_, launch2_.name as name8_8_2_, launch2_.number as number9_8_2_, launch2_.project_id as project10_8_2_, launch2_.rerun as rerun11_8_2_, launch2_.start_time as start_t12_8_2_, launch2_.status as status13_8_2_, launch2_.user_id as user_id14_8_2_, launch2_.uuid as uuid15_8_2_, attributes3_.launch_id as launch_i5_29_3_, attributes3_.id as id1_29_3_, attributes3_.id as id1_29_4_, attributes3_.key as key2_29_4_, attributes3_.launch_id as launch_i5_29_4_, attributes3_.system as system3_29_4_, attributes3_.item_id as item_id6_29_4_, attributes3_.value as value4_29_4_, statistics4_.launch_id as launch_i4_40_5_, statistics4_.s_id as s_id1_40_5_, statistics4_.s_id as s_id1_40_6_, statistics4_.s_counter as s_counte2_40_6_, statistics4_.item_id as item_id3_40_6_, statistics4_.launch_id as launch_i4_40_6_, statistics4_.statistics_field_id as statisti5_40_6_, statistics5_.sf_id as sf_id1_41_7_, statistics5_.name as name2_41_7_, testitem6_.item_id as item_id1_17_8_, testitem6_.code_ref as code_ref2_17_8_, testitem6_.description as descript3_17_8_, testitem6_.has_children as has_chil4_17_8_, testitem6_.has_retries as has_retr5_17_8_, testitem6_.has_stats as has_stat6_17_8_, testitem6_.last_modified as last_mod7_17_8_, testitem6_.launch_id as launch_i8_17_8_, testitem6_.name as name9_17_8_, testitem6_.parent_id as parent_10_17_8_, testitem6_.path as path11_17_8_, testitem6_.retry_of as retry_o12_17_8_, testitem6_.start_time as start_t13_17_8_, testitem6_.test_case_hash as test_ca14_17_8_, testitem6_.test_case_id as test_ca15_17_8_, testitem6_.type as type16_17_8_, testitem6_.unique_id as unique_17_17_8_, testitem6_.uuid as uuid18_17_8_, testitemre7_.result_id as result_i1_18_9_, testitemre7_.duration as duration2_18_9_, testitemre7_.end_time as end_time3_18_9_, testitemre7_.status as status4_18_9_, issueentit8_.issue_id as issue_id1_5_10_, issueentit8_.auto_analyzed as auto_ana2_5_10_, issueentit8_.ignore_analyzer as ignore_a3_5_10_, issueentit8_.issue_description as issue_de4_5_10_, issueentit8_.issue_type as issue_ty5_5_10_, issuetype9_.id as id1_7_11_, issuetype9_.hex_color as hex_colo2_7_11_, issuetype9_.issue_group_id as issue_gr6_7_11_, issuetype9_.locator as locator3_7_11_, issuetype9_.issue_name as issue_na4_7_11_, issuetype9_.abbreviation as abbrevia5_7_11_, issuegroup10_.issue_group_id as issue_gr1_6_12_, issuegroup10_.issue_group as issue_gr2_6_12_ from public.log log0_ left outer join attachment attachment1_ on log0_.attachment_id=attachment1_.id left outer join public.launch launch2_ on log0_.launch_id=launch2_.id left outer join item_attribute attributes3_ on launch2_.id=attributes3_.launch_id left outer join statistics statistics4_ on launch2_.id=statistics4_.launch_id left outer join statistics_field statistics5_ on statistics4_.statistics_field_id=statistics5_.sf_id left outer join public.test_item testitem6_ on log0_.item_id=testitem6_.item_id left outer join public.test_item_results testitemre7_ on testitem6_.item_id=testitemre7_.result_id left outer join public.issue issueentit8_ on testitemre7_.result_id=issueentit8_.issue_id left outer join public.issue_type issuetype9_ on issueentit8_.issue_type=issuetype9_.id left outer join public.issue_group issuegroup10_ on issuetype9_.issue_group_id=issuegroup10_.issue_group_id where log0_.id=$1
2021-08-20 09:19:05.430 UTC [2909] ERROR:  column attachment1_.creation_date does not exist at character 420
```

**To Reproduce**
Steps to reproduce you could find in the attached video. Thanks.
[2021-08-20_12-31-35.zip](https://github.com/reportportal/reportportal/files/7020636/2021-08-20_12-31-35.zip)

**Expected behavior**
We see our attachments in tests and don't see any errors in DB logs.

**Comment by evjlobanova** [comment]
Hello, please check the documentation regarding migration issue https://github.com/reportportal/documentation/blob/master/src/md/src/TipsAndTricks/MigrationIssue.md

Please try to perform described steps, and let us know about the results.

**Comment by hyunil-shin** [comment]
@v1-wizard 
It could be helpful to you.
I had the same issues (attachments error).
I added creation_date column myself.
```
$ docker exec -ti reportportal_postgres_1 psql -U rpuser -d reportportal

reportportal=# alter table attachment add column creation_date timestamptz;
```

And I repeated to remove dirty something.
```
UPDATE schema_migrations SET dirty = 'f' WHERE dirty = 't';
```

Finally, I can see attachments of launches executed after the fix.
I could not see attachments executed before the fix.
I think attachment is lost due to migration.

For Dirty database version XX, refer to 
https://github.com/reportportal/migrations/tree/develop/migrations
